src/types/router.rs

```diff
@@ -529,4 +561,20 @@ mod router_tests {
         assert_eq!(edges[0].to.get_uid(), "2");
         assert_eq!(edges[1].to.get_uid(), "3");
     }
+
+    /// Test get_total_distance
+    #[test]
+    fn test_get_total_distance() {
```

Comment:
Also test the invalid edge cases

Reply:
[18f1798](https://github.com/Arrow-air/lib-router/pull/11/commits/18f1798cdd885703926e6be1d7aaa486130e8870)